trollmoves/tests/test_movers.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2020 Adam.Dybbroe
+
+# Author(s):
+
+#   Adam.Dybbroe <a000680@c21856.ad.smhi.se>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test the movers
+"""
+
+from unittest.mock import patch
+
+
+@patch('netrc.netrc')
+def test_open_connection(netrc):
+    """Check getting the netrc authentication."""
+    from trollmoves.movers import FtpMover
+
+    username = 'myusername'
+    password = 'mypasswd'
+    account = None
+
+    origin = '/path/to/mydata/filename.ext'
+
+    def raise_my_error():
+        raise FileNotFoundError
+
+    netrc.side_effect = raise_my_error
```

Comment:
The side effect can be directly set to `FileNotFoundError`, no need to wrap it in a function.

Reply:
Hmm, this was the way I got it working, spent some time on this already :-(
This doesn't work:
```
E       netrc.side_effect = raise FileNotFoundError
E                           ^
E   SyntaxError: invalid syntax
```
How do you mean @pnuu ?

Reply:
just do
```
netrc.side_effect = FileNotFoundError
```
should work in principle